Nimble/Expectation.swift

```diff
@@ -1,5 +1,25 @@
 import Foundation
 
+func expectTo<T, U where U: Matcher, U.ValueType == T>(expression: Expression<T>, matcher: U, to: String) -> (Bool, FailureMessage) {
+    var msg = FailureMessage()
+    msg.to = to
+    let pass = matcher.matches(expression, failureMessage: msg)
+    if msg.actualValue == "" {
+        msg.actualValue = "<\(stringify(expression.evaluate()))>"
+    }
+    return (pass, msg)
+}
+
+func expectToNot<T, U where U: Matcher, U.ValueType == T>(expression: Expression<T>, matcher: U, toNot: String) -> (Bool, FailureMessage) {
+    var msg = FailureMessage()
+    msg.to = toNot
+    let pass = matcher.doesNotMatch(expression, failureMessage: msg)
+    if msg.actualValue == "" {
+        msg.actualValue = "<\(stringify(expression.evaluate()))>"
+    }
+    return (pass, msg)
+}
```

Comment:
I guess the point of both of these functions is to call `matches` or `doesNotMatch`, then check if any value has been set for the `actualValue` message. If none has been set, the functions set a default value. That right?

A nitpick to be sure, but since I had to read the implementation to grok the point of the functions, I'd try:

1. Rename these two functions to express the above intent
1. Extract the shared logic into a function with an expressive name (on the condition that made the code easier to read)
1. Add comments if the first two steps haven't cleared this up

Up to you! :basketball: